### PR TITLE
fix(PausableImage): Update to better handle svg height

### PR DESCRIPTION
**Pull request by vonbarown**
### PR Checklist

- [ ] Related to designs:
- [ ] Related to JIRA ticket: ABC-123
- [ ] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.79-alpha.55a212.0<br/>@codecademy/gamut-labs@30.2.5-alpha.55a212.0<br/>@codecademy/styleguide@55.1.5-alpha.55a212.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://622fa3e1f920c800b5d0ccd0--gamut-preview.netlify.app>https://622fa3e1f920c800b5d0ccd0--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/622fa3e1f920c800b5d0ccd0>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.79-alpha.8c3842.0<br/>@codecademy/gamut-labs@30.2.5-alpha.8c3842.0<br/>@codecademy/styleguide@55.1.5-alpha.8c3842.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://622fb0323da6a41594c6b117--gamut-preview.netlify.app>https://622fb0323da6a41594c6b117--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/622fb0323da6a41594c6b117>Deploy Logs</a></p>

File: packages/gamut-labs/src/PausableImage/index.tsx [hunk 1]
```diff
@@ -13,11 +13,12 @@ interface PauseableImageProps {
 }
 export const imageStyles = styled.img(
   css({
-    height: 'auto',
+    height: '100%',
     maxHeight: '100%',
     maxWidth: '100%',
     '&[src$=".svg"]': {
       width: '100%',
+      heigh: '100%',
```
Comment (dreamwasp) [inline packages/gamut-labs/src/PausableImage/index.tsx:21]:
```suggestion
      height: '100%',
```

**Review by dreamwasp — CHANGES_REQUESTED** [review]

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-kit@0.6.79-alpha.873183.0<br/>@codecademy/gamut-labs@30.2.5-alpha.873183.0<br/>@codecademy/gamut@45.2.0-alpha.873183.0<br/>@codecademy/monaco@0.6.25-alpha.873183.0<br/>@codecademy/styleguide@55.1.5-alpha.873183.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://62309bcfde67ff183840c987--gamut-preview.netlify.app>https://62309bcfde67ff183840c987--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/62309bcfde67ff183840c987>Deploy Logs</a></p>

File: packages/gamut-labs/src/PausableImage/index.tsx (hunk 1 above)
Comment (vonbarown) [reply packages/gamut-labs/src/PausableImage/index.tsx:21]: thanks for catching, the issue doesn't appear to be from the gamut component but from the container on the monolith. have a fix in this [PR](https://github.com/codecademy-engineering/Codecademy/pull/29919).

**Pull request closed without merging**